.github/workflows/eks.yaml

```diff
@@ -102,8 +103,9 @@ jobs:
             --set tag=${{ steps.vars.outputs.sha }} \
             --set cluster_name=${{ env.clusterName }}
 
-      - name: Wait for test job
+      - name: Wait for and copy the sysdump in the background and wait for job termination
         run: |
+          kubectl -n kube-system exec job/cilium-cli -- cat /tmp/cilium-sysdump-out > cilium-sysdump-out.zip &
```

Comment:
I have had all kinds of trouble trying to use `exec` in gh workflows before, sometimes connections are dropping and this would probably cause sysdumps to be malformed. Let's see how this works, the alternative to this approach is to put the sysdump in a configmap inside cli pod and retrieve it via `kubectl get` in workflow later (like we do with vm install script in external workloads workflow).

Reply:
That won't work unfortunately — configmaps are limited to 1MB 😞 We'll have to resort to PVCs in that case.